cpg_workflows/dataproc_scripts/mt_no_dp_wrapper.py

```diff
@@ -0,0 +1,30 @@
+from argparse import ArgumentParser
+
+from cpg_utils.config import config_retrieve
+from cpg_utils.hail_batch import get_batch
+
+
+def main():
+    parser = ArgumentParser(description='Argument Parser for the ES generation script')
+    parser.add_argument('--mt_path', help='MT path name', required=True)
+    parser.add_argument('--index', help='ES index name', required=True)
+    parser.add_argument('--flag', help='ES index "DONE" file path')
+    args = parser.parse_args()
+
+    job = get_batch().new_job(f'Generate {args.index} from {args.mt_path}')
+    job.image(config_retrieve(['workflow', 'driver_image']))
+    job.storage(config_retrieve(['workflow', 'storage_requirement'], '10Gi'))
```

Comment:
Do we need to make this dynamic even if it is a dirty heuristic based on sample size?

Reply:
Should be easy enough to squeeze that in if we proceed to use this. We'd be running in a pipeline workflow context, so cohort size & exome/genome will be available info